src/main/java/uk/gov/justice/digital/zone/RawZone.java

```diff
@@ -30,6 +34,11 @@ public class RawZone extends Zone {
     private final DataStorageService storage;
     private final SourceReferenceService sourceReferenceService;
 
+    private final Set<String> cdcOperationsSet = Arrays
```

Comment:
I think it'd be good to have this on the `Operation` enum. So we could have something like `getCdcOperations` that returns a set. 

The reasoning being that this way, we only need to define this information once, and the definition is located right next to the definiton of the values themselves.